tools/ci_build/builds/release_macos.sh

```diff
@@ -18,43 +18,27 @@ set -e -x
 # No GPU support for MacOS
 export TF_NEED_CUDA="0"
 
-PYTHON_VERSIONS="3.5.6 3.6.6 3.7.4"
```

Comment:
For the future usages: where are specified the target python versions for the wheels now?

Reply:
My bad, it's in the github actions file.